refinedstorage-neoforge/src/test/java/com/refinedmods/refinedstorage/common/storage/diskinterface/DiskInterfaceTestPlots.java

```diff
@@ -0,0 +1,96 @@
+                            final TriConsumer<AbstractDiskInterfaceBlockEntity, BlockPos, GameTestSequence> consumer) {
+        helper.setBlock(ZERO.above(), RSBLOCKS.getCreativeController().getDefault());
+        helper.setBlock(ZERO.above().above(), RSBLOCKS.getItemStorageBlock(ItemStorageVariant.ONE_K));
+        final BlockPos diskInterfacePos = ZERO.above().above().above();
+        helper.setBlock(diskInterfacePos, RSBLOCKS.getDiskInterface().getDefault()
+            .rotated(BiDirection.forDirection(direction)));
+        consumer.accept(
+            requireBlockEntity(helper, diskInterfacePos, AbstractDiskInterfaceBlockEntity.class),
+            diskInterfacePos,
+            helper.startSequence()
+        );
+    }
+
+    static void addDiskToDiskInterface(final GameTestHelper helper,
+                                       final BlockPos diskInterfacePos,
+                                       final ResourceAmount... resources) {
+        final ItemStack diskItem = new ItemStack(Items.INSTANCE.getItemStorageDisk(ItemStorageVariant.SIXTY_FOUR_K));
+        diskItem.inventoryTick(helper.getLevel(), helper.makeMockPlayer(GameType.SURVIVAL), 0, false);
+
+        final SerializableStorage storage = getSerializableStorageFromDisk(helper, diskItem);
+        if (resources.length > 0) {
+            final PlayerActor actor = new PlayerActor(helper.makeMockPlayer(GameType.SURVIVAL));
+            for (final ResourceAmount resource : resources) {
+                if (resource != null) {
+                    storage.insert(resource.resource(), resource.amount(), Action.EXECUTE, actor);
+                }
+            }
+        }
+
+        final var diskInterfaceBlockEntity = requireBlockEntity(helper, diskInterfacePos,
+            AbstractDiskInterfaceBlockEntity.class);
+        final FilteredContainer diskInterfaceContainer = diskInterfaceBlockEntity.getDiskInventory();
+        diskInterfaceContainer.addItem(diskItem);
+    }
+
+    static void isDiskInOutputWithAmount(final GameTestHelper helper,
+                                         final BlockPos diskInterfacePos,
+                                         final int storedAmount) {
+        final var diskInterfaceBlockEntity = requireBlockEntity(helper, diskInterfacePos,
+            AbstractDiskInterfaceBlockEntity.class);
+        final FilteredContainer diskInterfaceContainer = diskInterfaceBlockEntity.getDiskInventory();
+
+        final ItemStack diskItem = diskInterfaceContainer.getItem(3);
+        final SerializableStorage storage = getSerializableStorageFromDisk(helper, diskItem);
+
+        helper.assertTrue(!diskItem.isEmpty(), "Could not find a Storage Disk in the output slot");
+        helper.assertTrue(storage.getStored() == storedAmount, "Expected Storage Disk stored amount "
+            + storedAmount + ", but was " + storage.getStored());
+    }
+
+    private static SerializableStorage getSerializableStorageFromDisk(final GameTestHelper helper,
```

Comment:
```suggestion
    private static Storage getStorageFromDisk(final GameTestHelper helper,
```
Downcast to Storage as it doesn't matter that it's serializable, also reflect it in the method name.